packages/office-ui-fabric-react/src/utilities/dragdrop/DragDropHelper.tsx

```diff
@@ -285,8 +304,7 @@ export class DragDropHelper implements IDragDropHelper {
         let xDiff = this._dragData.clientX - event.clientX;
         let yDiff = this._dragData.clientY - event.clientY;
         if (xDiff * xDiff + yDiff * yDiff >= DISTANCE_FOR_DRAG_SQUARED) {
-          if (this._dragData.dragTarget &&
-            this._selection.isIndexSelected(options.selectionIndex)) {
```

Comment:
Ok so we decided to revert this change. I think this should still work with your scenario with the marquee selection.